Modules/Devices/src/Devices.Application/Infrastructure/Persistence/Repository/IIdentitiesRepository.cs

```diff
@@ -1,10 +1,26 @@
 ﻿using Backbone.Modules.Devices.Domain.Entities;
 using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.Database;
 using Enmeshed.BuildingBlocks.Application.Pagination;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
 
 namespace Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
 
 public interface IIdentitiesRepository
 {
+    #region Identities
     Task<DbPaginationResult<Identity>> FindAll(PaginationFilter paginationFilter);
+#nullable enable
+    Task<Identity?> FindByAddress(IdentityAddress address, CancellationToken cancellationToken);
+#nullable disable
+    #endregion
+
+    #region Users
+    Task AddUserForIdentity(ApplicationUser user, string password);
+    #endregion
+
+    #region Devices
+    Task<DbPaginationResult<Device>> FindAll(IdentityAddress identity, IEnumerable<DeviceId> ids, PaginationFilter paginationFilter);
```

Comment:
I don't like that it has the same name as the method for retrieving all Identities. 

```suggestion
    Task<DbPaginationResult<Device>> FindAllDevicesOfIdentity(IdentityAddress identity, IEnumerable<DeviceId> ids, PaginationFilter paginationFilter);
```